### PR TITLE
Sign in and Sign up form

### PR DESCRIPTION
**Pull request by yanlifang**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -41,3 +44,6 @@
     -->
   </body>
 </html>
+<!-- Compiled and minified JavaScript -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```
Comment (dlimx) [inline public/index.html:48]: Let's pick one of materialize and bootstrap

File: src/index.js [hunk 2]
```diff
@@ -1,21 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
+
 import './index.css';
 import App from './components/App';
-import * as serviceWorker from './serviceWorker';
```
Comment (dlimx) [inline src/index.js:5]: We can delete this file then

File: package.json [hunk 3]
```diff
@@ -30,13 +30,15 @@
     "react-bootstrap": "^1.1.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "redux-form": "^8.3.6"
```
Comment (dlimx) [inline package.json:34]: What is `redux-form` being used for?

As a side note I'd recommend `formik` but plain React forms work

File: src/index.js (hunk 2 above)
Comment (yanlifang) [reply src/index.js:5]: Ok, I am not sure if we need this in the future. I did not delete it.

File: package.json (hunk 3 above)
Comment (yanlifang) [reply package.json:34]: OK. I used some labels told me to install redux-form, but I don't think I use it.  I can delete this.

File: public/index.html (hunk 1 above)
Comment (yanlifang) [reply public/index.html:48]: I will delete this materialize.

**Pull request merged**